CHANGELOG.md

```diff
@@ -2,6 +2,48 @@
 
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.7.1] - (21/08/19) - O iOS
+
+### Features
+- Voice features STT & TTS can now be turned off
+
+---
+
+## [1.7.0] - (5/07/19) - I iOS
```

Comment:
How is this not in `master` already?

Reply:
The changelog was being maintained on the Release Tab Notes @tejas-haptik. Will be updating it from now on.

Reply:
Great. Thanks @iamsimranjot